libs/oidc/common/src/lib/modules/auth/auth.module.ts

```diff
@@ -0,0 +1,21 @@
+import { DynamicModule, Module } from '@nestjs/common';
+
+import { JwtStrategy } from '../../strategies/jwt/jwt.strategy';
+import { JWKS_URL } from '../tokens/jwks_url.token';
+
+@Module({})
+export class AuthModule {
+  public static forRoot(options: { jwksUrl: string }): DynamicModule {
```

Comment:
Is the return type here required? You don't usually include that stuff.

Reply:
It's not, but it makes it so the return object is typed whenever I am writing it out. `forRoot`, `forChild`, `register`, w/e you want to call initialization static methods can have whatever name, it just comes down to semantics. And since these methods can return anything and there isn't any interface `implements`, then the IDE can't know that it's supposed to return  (like OnInit that returns `void` and returning anything else violates the signature.)